### PR TITLE
Added logger for project

### PR DESCRIPTION
**Pull request by shtripat**
To use this utility, initialize the logger as below first

```
import (
	"github.com/gluster/gluster-prometheus/pkg/logging"

	log "github.com/sirupsen/logrus"
)

if err := logging.Init({sample-log-dir}, {sample-log-file}, {sample-log-level}); err != nil {
        log.WithError(err).Fatal("Failed to initialize logging")
}
```

After initialize done for logging the message use below code

```
log.WithError({err-object}).Fatal("{error-message}")
```
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

File: pkg/logging/logger.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package logging
+
+import (
+        "io"
+        stdlog "log"
+        "os"
+        "path"
+        "strings"
+
+        log "github.com/sirupsen/logrus"
```
Comment (aravindavk) [inline pkg/logging/logger.go]: please update `Gopkg.toml` for this new dependency

Comment (shtripat) [reply pkg/logging/logger.go]: Ack

**Comment by shtripat** [comment]
I will add a sample code for using the same from main.go

**Comment by aravindavk** [comment]
Fixes: #20

**Comment by aravindavk** [comment]
Please add `Gopkg.lock` file similar to config PR.

**Pull request merged**